pkg/net/ip_port.go

```diff
@@ -16,21 +16,31 @@ type NetworkPeerID struct {
 	IPNetwork IPNetwork
 }
 
-// IsValid checks if the ip:port pair is valid.
-func (e NetworkPeerID) IsValid() bool {
+// IsAddressValid checks if the ip:port pair of the Address is valid.
```

Comment:
[nit] I think that the port is not checked

Reply:
I only renamed this function to be less confusing. Are you referring to the comment that it checks `ip:port` pair?

Reply:
> Are you referring to the comment that it checks ip:port pair?

yes

Reply:
Based on the unit tests, this function is expected to only check the IP address and skip the port. So I changed the comment to reflect that behavior in [98ab253](https://github.com/stackrox/stackrox/pull/9718/commits/98ab253fdbbc270a5d5c112829ff08fe57428f1e)